css/style.css

```diff
@@ -61,19 +59,213 @@ body {
 
 /* Center the container */
 .container {
-  width: 960px;
-  margin: auto;
-  height: 500px;
+  width: 100%;
+  margin: 0 auto;
+  height: 600px;
```

Comment:
@jonpackles You should take the height off the container, you generally let the content dictate the height of an element.